### PR TITLE
Created specific ldmd.conf files for our separate EDEX systems (main, radar, sat) so we weren't pulling more data than needed

### PR DESCRIPTION
**Pull request by tiffanycmeyer13**

### CODE REVIEW

File: rpms/awips2.upc/Installer.ldm/patch/etc/ldmd.goesr [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+# of data-products (i.e., data-products will be rapidly sent using the
+# HEREIS message) and the other hosts will initially be alternate sources of
+# data-products (i.e., data will be sent using the COMMINGSOON and BLKDATA
+# messages).  The primary host will probably change over time -- depending on
+# which host can deliver the data-products most quickly on average.
+#
+#
+# DEFAULT - Satellite Ancillary EDEX
+#
+REQUEST NIMAGE "CloudAndMoistureImagery" 10.0.0.233 # GOES Stiched Channels (Unidata)
```
Comment (srcarter3) [inline rpms/awips2.upc/Installer.ldm/patch/etc/ldmd.goesr]: Do we want these to have what our users should be requesting from? `idd.unidata.ucar.edu`
And then we obviously modify them to our internal relays when we use them?

Comment (tiffanycmeyer13) [reply rpms/awips2.upc/Installer.ldm/patch/etc/ldmd.goesr]: oh yeah, you're right

Comment (srcarter3) [reply rpms/awips2.upc/Installer.ldm/patch/etc/ldmd.goesr]: Okay cool, my only other question. I thought previously we had some of our public (and maybe internal as well) entries pointing at `hrrr.unidata.ucar.edu`, is that no longer the case?

Comment (tiffanycmeyer13) [reply rpms/awips2.upc/Installer.ldm/patch/etc/ldmd.goesr]: No longer the case. Mike has all of of that data going through idd now and actually requested that we request from there and not hrrr.unidata.ucar.edu

Comment (srcarter3) [reply rpms/awips2.upc/Installer.ldm/patch/etc/ldmd.goesr]: Okay great. Ya I thought that had changed but couldn't remember. Thanks!

**Pull request merged**